frontend/public/style/_overrides.scss

```diff
@@ -1,5 +1,4 @@
 // Use this file to override styles from 3rd party dependencies
-$pf-4-nav-bar-height: 76px; // Height of the PatternFly 4 masthead
```

Comment:
This really should have gone with https://github.com/openshift/console/pull/9452 as it is orphaned, but @spadgett already tagged that and I don't want him to have to retag it.